docs/protocol/Validator Sets.md

```diff
@@ -0,0 +1,28 @@
+# Validator Sets
+
+Validator Sets are defined at the protocol level, with the following parameters:
+
+- `index` (VS):        Validator set index, a global key atomically increasing
+from 0.
+- `bond`  (Amount):    Amount of bond per key-share of this validator set.
+- `coins` (Vec<Coin>): Coins managed by this validator set.
+
```

Comment:
I'd rather simply append in hard forks, and if we ever remove for some reason, set `coins.len` to `0`. I don't see a benefit for an additional status as a protocol level, especially since rewards would be based on their managed coins (encouraging validators to provide bond where users need it), so no coins would cause it to have no effect. Seems more like an RPC comment. Feel free to clarify why though :)